tests/Handler/CurlFactoryTest.php

```diff
@@ -65,11 +65,11 @@ public function testCreatesCurlHandle()
                 $_SERVER['_curl'][CURLOPT_PROTOCOLS]
             );
         }
-        self::assertContains('Expect:', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
-        self::assertContains('Accept:', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
-        self::assertContains('Content-Type:', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
-        self::assertContains('Hi: 123', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
-        self::assertContains('Host: 127.0.0.1:8126', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
+        self::assertStringContainsString('Expect:', $_SERVER['_curl'][CURLOPT_HTTPHEADER]);
```

Comment:
The second parameter here must be a string.  `$_SERVER['_curl'][CURLOPT_HTTPHEADER]` is an array.